website/content/en/preview/getting-started/migrating-from-cas/_index.md

```diff
@@ -16,59 +16,85 @@ We will make the following assumptions in this guide
 * Your workloads have pod disruption budgets that adhere to [EKS best practices](https://aws.github.io/aws-eks-best-practices/karpenter/)
 * Your cluster has an [OIDC provider](https://docs.aws.amazon.com/eks/latest/userguide/enable-iam-roles-for-service-accounts.html) for service accounts
 
-This guide will also assume you have the `aws` CLI installed.
+This guide will also assume you have the `aws` CLI and Helm installed.
 You can also perform many of these steps in the console, but we will use the command line for simplicity.
 
-Set a variable for your cluster name.
+## Set environment variables
+Set the Karpenter and Kubernetes version. Check the [Compatibility Matrix](https://karpenter.sh/docs/upgrading/compatibility/) to find the Karpenter version compatible with your current Amazon EKS version.
 
 ```bash
-KARPENTER_NAMESPACE=kube-system
+KARPENTER_NAMESPACE="kube-system"
+KARPENTER_VERSION="{{< param "latest_release_version" >}}"
+K8S_VERSION="{{< param "latest_k8s_version" >}}"
 CLUSTER_NAME=<your cluster name>
 ```
 
 Set other variables from your cluster configuration.
 
 {{% script file="./content/en/{VERSION}/getting-started/migrating-from-cas/scripts/step01-env.sh" language="bash" %}}
 
-Use that information to create our IAM roles, inline policy, and trust relationship.
+{{% alert title="Warning" color="warning" %}}
+If you open a new shell to run steps in this procedure, you need to set the environment variables again.
+{{% /alert %}}
 
 ## Create IAM roles
 
-To get started with our migration we first need to create two new IAM roles for nodes provisioned with Karpenter and the Karpenter controller.
-
-To create the Karpenter node role we will use the following policy and commands.
-
-{{% script file="./content/en/{VERSION}/getting-started/migrating-from-cas/scripts/step02-node-iam.sh" language="bash" %}}
+Use CloudFormation to set up the infrastructure needed by the existing EKS cluster. See [CloudFormation]({{< relref "../../reference/cloudformation/" >}}) for a complete description of what `cloudformation.yaml` does for Karpenter. The provided `cloudformation.yaml` template simplifies this setup by creating and configuring all necessary resources, including:
```

Comment:
Personally, I prefer the use of `aws` commands over cloudformation, as I'd end up doing it all with terraform anyway but not sure what other things or what the general opinion on this is.
That said, you have a point here in that the cloudformation will script will also setup the interruption queue and policies, which are later used but never setup in the current version of this document.

Reply:
That's correct.